freqtrade/freqtradebot.py

```diff
@@ -898,7 +898,7 @@ def handle_timedout_limit_buy(self, trade: Trade, order: Dict) -> bool:
         Buy timeout - cancel order
         :return: True if order was fully cancelled
         """
-        if order['status'] != 'canceled':
+        if order['status'] not in ('canceled', 'closed'):
```

Comment:
```suggestion
        # Cancelled orders may have the status of 'canceled' or 'closed' (ftx)
        if order['status'] not in ('canceled', 'closed'):
```
-- in the future, this can probably be additionally 'unified' by moving this exchange-specific code in the exchange-specific freqtrade Exchange class...

Reply:
I did explicitly avoid using FTX in the comment here as other exchanges will most likely use the same pattern (ftx is not THAT unique). 
Also, this is not the only place where we check for canceled and closed to determine if an order was cancelled :)

Reply:
I meant here, a comment is better be added... Will it contain ftx or not -- is the second point...